composer.json

```diff
@@ -35,12 +36,13 @@
     },
     "require-dev": {
         "beyondcode/expose": "^2.0",
+        "friendsofphp/php-cs-fixer": "!=3.5.0",
         "johnkary/phpunit-speedtrap": "^4.0",
         "phlak/coding-standards": "^2.0",
-        "friendsofphp/php-cs-fixer": "!=3.5.0",
         "phpstan/phpstan": "^1.0",
         "psy/psysh": "^0.11.0",
         "symfony/var-dumper": "^5.0",
+        "timeweb/phpstan-enum": "^3.1",
```

Comment:
Please add additional packages.

```suggestion
```

Reply:
This package is required to allow PHP Stan to properly evaluate the Enums used in ZipStream (they use it for archive types, for instance). So, I can either use this package to make it aware of them, or ignore the lines (but this is not the suitable option in my opinion).

Reply:
Which enums are we talking about? Enums weren't introduced until PHP 8.1 and as discussed in the issue I'd like to keep support for PHP 7.4 for now.

Reply:
This is a custom Enum module compatible with any version of PHP called php-enum. The ZipStream library uses them here: https://github.com/maennchen/ZipStream-PHP/blob/v2/src/Option/Method.php
Since this is dynamically evaluated, PHPStan is all lost when seeing "unexisting methods" being called.
In the latest version (v3), since they killed the compat with older versions of PHP, they moved to native enums: https://github.com/maennchen/ZipStream-PHP/blob/main/src/CompressionMethod.php

I am making a usage of those enums in the code to indicate the type of compression to apply here: https://github.com/DirectoryLister/DirectoryLister/blob/5b61653b889cec276bdc6bc0cd45a66e6aafdac9/app/src/Controllers/ZipController.php#L62